ouroboros-network/test/Test/Ouroboros/Network/Diffusion/Node/MiniProtocols.hs

```diff
@@ -287,6 +316,14 @@ applications debugTracer nodeKernel
                     (keepAliveInitiator connId controlMessageSTM)
                     keepAliveResponder
               }
+          , MiniProtocol
+              { miniProtocolNum    = MiniProtocolNum 12
```

Comment:
If we change the protocol number to `10` we should update this.  It's better to use `Ouroboros.Network.NodeToNode.peerSharingMiniProtocolNum` (define it if it isn't).